gradle-module/src/main/java/org/example/Main.java

```diff
@@ -5,7 +5,8 @@ public class Main {
     public static final String CONSTANT = "constant";
 
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        String s = "Hello world!";
+        System.out.println(s);
     }
 
```

Comment:
The change from a direct string literal to a variable is unnecessary in this context. If the string is not reused, consider keeping the original implementation for simplicity. This change adds unnecessary complexity without providing a clear benefit.